alerter.js

```diff
@@ -1,23 +1,30 @@
+const { expect } = require('chai');
+
 let alertFailureCount = 0;
 
 function networkAlertStub(celcius) {
-    console.log(`Alert! Temperature is ${celcius} degrees`);
-    // Return 200 for ok
-    // Return 500 for not-ok
-    // stub always succeeds and returns 200
-    return 200;
+	console.log(`Alert! Temperature is ${celcius} degrees`);
+	// Return 200 for ok
+	// Return 500 for not-ok
+	// stub always succeeds and returns 200
+	return 200;
 }
 
 function alertInCelcius(farenheit) {
-    const celcius = (farenheit - 32) * 5 / 9;
-    const returnCode = networkAlertStub(celcius);
-    if (returnCode != 200) {
-        // non-ok response is not an error! Issues happen in life!
-        // let us keep a count of failures to report
-        // However, this code doesn't count failures!
-        // Add a test below to catch this bug. Alter the stub above, if needed.
-        alertFailureCount += 0;
-    }
+	const celcius = convertFarenheitToCelcius(farenheit);
+	const returnCode = networkAlertStub(celcius);
+	countAlertFailure(returnCode);
+}
+
+function convertFarenheitToCelcius(farenheit) {
+	let celcius = ((farenheit - 32) * 5) / 9;
+	return celcius;
+}
+
+function countAlertFailure(code) {
+	if (code !== 200) {
+		return (alertFailureCount += 1);
+	}
 }
 
 alertInCelcius(400.5);

```

Comment:
there is no test case that asserts or verifies the intended return value of production code. After alertInCelcius() is called , the error count needs to be verified. Also the production code needs to be updated to return 'ok' or 'not ok' response code based on temperature value.